### DISCUSSION TITLE
[logger][HPARAMS] I do not see anything in the TensorBord, in the HPARAMS tab. pytorch-lightning==1.3.0rc1

### DISCUSSION THREAD

**Discussion by paantya** [Lightning-AI/pytorch-lightning#7124] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Since we started about the parameters

I use, and I do not see anything in the TensorBord, in the HPARAMS tab.

```
class QLearning(pl.LightningModule):
        ...
        super().__init__()
        self.save_hyperparameters(ignore=['env', 'env_valid'])

    # Using custom or multiple metrics (default_hp_metric=False)
    def on_train_start(self):
        print(f"self.hparams: {self.hparams}")
        self.logger.log_hyperparams(self.hparams, metrics={'hp_metric': 0})
        # self.logger.log_hyperparams(self.hparams,)
```

i try with default_hp_metric=False, default_hp_metric=True, the result is the same, nothing appears.
```
tb_logger = pl_loggers.TensorBoardLogger(
    save_dir='lightning_logs/',
    name="QLearning",
    version=None,
    log_graph=False,
    default_hp_metric=False,
    # prefix="test",
)

pl_trener = pl.Trainer(
    max_epochs=20*1000,
    logger=tb_logger,
)
```

maybe I'm doing something wrong?

requirements.txt:

atari-py==0.2.6
gym==0.18.0
h5py==2.10.0
hydra-core==1.0.6
joblib==1.0.1
kaggle==1.5.10
matplotlib==3.3.4
multidict==5.1.0
numpy==1.19.5
omegaconf==2.0.6
pytorch-lightning==1.3.0rc1
pytorch-lightning-bolts==0.3.0
PyYAML==5.3.1
redis==3.5.3
tensorboard==2.4.1
tensorboard-plugin-wit==1.8.0
tensorboardX==2.1
tensorflow-estimator==2.4.0
tensorflow-gpu==2.4.1
torch==1.7.1
torchvision==0.8.2
tqdm==4.56.0
urllib3==1.26.3
opencv-python==4.5.1.48
opencv-python-headless==4.5.1.48

setuptools~=46.1.3
Cython~=0.29.16
pandas~=1.0.3
bs4~=0.0.1
beautifulsoup4~=4.9.3
scikit-learn~=0.24.1

**Comment by paantya** [comment]
I restarted everything I could, deleted the old logs and it seemed to work.

**Reply by awaelchli** [reply]
It's possible that TensorBoards updates these only at the end of the run. It was like this a while ago, not sure if that changed in the mean time. It could explain why you didn't see them first.

**Reply by Big-Tree** [reply]
Deleting the old logs also solved the issue for me.

**Reply by paantya** [reply]
+1
deleted magazines + updated tb, and fix it
